app/services/budget/export.rb

```diff
@@ -52,10 +55,17 @@ def activity_data(activity)
         activity.partner_organisation_identifier,
         activity.extending_organisation&.name,
         I18n.t("table.body.activity.level.#{activity.level}"),
-        activity.title
+        activity.title,
+        activity.programme? ? activity_comments(activity) : ""
       ]
     end
 
+    def activity_comments(activity)
```

Comment:
As you mentioned in your PR description, we don't test private methods explicitly. Instead we test them indirectly via the methods they're called e.g. we're covering this with tests for `rows`.